packages/functional-tests/pages/signupReact.ts

```diff
@@ -12,46 +12,67 @@ export class SignupReactPage extends BaseLayout {
     );
   }
 
+  get emailFirstHeader() {
+    return this.page.getByText(/Enter your email/);
+  }
+
   get passwordHeader() {
     return this.page.getByText('Set your password');
   }
 
-  setEmail(value: string) {
-    return this.page.fill('[name="email"]', value);
+  get confirmCodeHeading() {
```

Comment:
Is this used? I don't see any references to it when search the code.

Reply:
Ditto.